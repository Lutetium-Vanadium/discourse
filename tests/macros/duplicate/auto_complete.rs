fn main() {
    discourse::questions![Input {
        auto_complete: todo!(),
        auto_complete: todo!()
    }];
}
