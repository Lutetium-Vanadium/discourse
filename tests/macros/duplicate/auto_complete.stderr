error: duplicate option `auto_complete`
 --> $DIR/auto_complete.rs:4:9
  |
4 |         auto_complete: todo!()
  |         ^^^^^^^^^^^^^
